src/sensors.c

```diff
@@ -155,7 +160,9 @@ void init_sensors(void)
   mpu6050_register_interrupt_cb(&imu_ISR);
 
   uint16_t acc1G;
-  mpu6050_init(true, &acc1G, &gyro_scale, _params.values[PARAM_BOARD_REVISION]);
+  float gyro_scale_to_urad;
+  mpu6050_init(true, &acc1G, &gyro_scale_to_urad, _params.values[PARAM_BOARD_REVISION]);
+  gyro_scale = 1000000.0*gyro_scale_to_urad;
```

Comment:
Haha yeah, that's the weird thing about stuff like this. You find a bug that seems like it should be completely wrong, then you fix it and nothing changes. Happens to me all the time... no idea why it's like that.

So I guess the question now is will it work the "right" way?... It could be that it's actually right and we're misunderstanding what the gyro scale that `mpu6050_init` returns is doing in the first place. Or it could be that it is wrong but it can be compensated for with gains and whatnot. Difficult to say without some testing.